.vscode/settings.dist.jsonc

```
@@ -1,41 +1,40 @@
+// This is a managed VS Code settings file.
+// It is copied into place by `tools/install-vscode-settings.sh`.
+// If you no longer want the settings managed, remove these comments.
 {
-	"editor.codeActionsOnSave": {
-		"source.fixAll.eslint": "explicit"
-	},
+	// Don't show these files in VS Code.
```

I would suggest adding the JSON schema to this file.

```suggestion
	"$schema": "vscode://schemas/settings/user",
	// Don't show these files in VS Code.
```

---

Great idea! I used `vscode://schemas/settings/workspace` instead, as technically it's [workspace settings](https://github.com/wraith13/vscode-schemas/blob/master/en/latest/schemas/settings/workspace.json) (unless it's really [folder settings](https://github.com/wraith13/vscode-schemas/blob/master/en/latest/schemas/settings/folder.json)...VS Code is rather unclear on this, but for now, they're identical).

2fb6a5321f

---

It's just the schema reference, shouldn't be much different.

---

There are 900 lines of difference in the [user settings](https://github.com/wraith13/vscode-schemas/blob/master/en/latest/schemas/settings/user.json) schema. Not that it'll matter in a practical sense, but doesn't hurt to be precise. :^)